packages/ses/package.json.md

```diff
@@ -98,6 +98,9 @@ The variations differ only in file name extension.
 
 Node.js and other tools will use this file when importing `ses` as an ESM.
 (JavaScript module).
+We have in the past experimented with using the precompiled bundle of SES here
+(`./dist/ses.cjs`), but found that this interacted poorly with Endo,
```

Comment:
We were using the UMD version, not the CommonJS version.

```suggestion
(`./dist/ses.umd.js`), but found that this interacted poorly with Endo,
```

Reply:
Addressed with “(`./dist/ses.cjs` or `./dist/ses.umd.js`)” since these files are identical)